context.go

```diff
@@ -109,31 +110,42 @@ func CreateContext(masterKey, masterSalt []byte, profile ProtectionProfile, opts
 
 // https://tools.ietf.org/html/rfc3550#appendix-A.1
 func (s *srtpSSRCState) nextRolloverCount(sequenceNumber uint16) (uint32, func()) {
-	roc := s.rolloverCounter
-
-	switch {
-	case !s.rolloverHasProcessed:
-	case sequenceNumber == 0: // We exactly hit the rollover count
-		// Only update rolloverCounter if lastSequenceNumber is greater then maxROCDisorder
-		// otherwise we already incremented for disorder
-		if s.lastSequenceNumber > maxROCDisorder {
-			roc++
+	seq := int32(sequenceNumber)
+	localRoc := uint32(s.index >> 16)
+	localSeq := int32(s.index & 0xffff)
```

Comment:
```suggestion
	localSeq := int32(s.index & (seqNumMax - 1))
```
just to make it consistent with https://github.com/pion/srtp/pull/146/files#diff-552f47512a00afe5fc6850cc9ddc830a6daeca162750e50aab4ed549685e0253R192
(`seqNumMax - 1` is evaluated during compilation)